protocols/migration/migration_reports_400_to_reports_500.py

```diff
@@ -163,6 +164,9 @@ def migrate_cancer_interpretation_request_to_cancer_interpreted_genome(
         new_instance.interpretationRequestVersion = old_instance.reportVersion
         new_instance.interpretationService = interpretation_service
         new_instance.referenceDatabasesVersions = reference_database_versions
+        if software_versions is None:
```

Comment:
maybe something like `if not isinstance(software_versions, dict):` just in case the method is passed something like `software_versions=""` ?